usaspending_api/awards/models.py

```diff
@@ -960,3 +960,16 @@ class Subaward(DataSourceTrackedModel):
     class Meta:
         managed = True
         db_table = 'subaward'
+
+
+class ParentAward(models.Model):
+    award = models.OneToOneField(Award, primary_key=True)
```

Comment:
Because we're keeping dupes "as is" for now, we can't use award_id as the primary key.  Some options are

1. no primary key - Django doesn't like this
2. surrogate primary key - just another dumb number, but more space efficient than # 3
3. primary key that is a list of [award_id, parent_award_id] - don't know if it's worth the hassle

Thoughts?